python/modules/map_plate.py

```diff
@@ -41,95 +76,51 @@ def _getting_paths_mp(input_path):
         if path not in subfile_list and path[-4:] == ".csv":
             param_paths.append(path)
     return param_paths
-    
-def _making_path_list(main_path, file_list): # maybe this should be in file_managment?? gonna change method of path joining to more safe (not simple strings merge)
-    path_list = []
-    for filepath in file_list:
-        path = main_path + filepath
-        path_list.append(path)
-    return path_list
 
-def _parsing_matrix(path, deltimer, exp_part):
-    active_wells = []
-    data = CSV_M.read_csv(path, deltimer)
-    for row in range(len(data)):
-        for col in range(len(data[row])):
-            if data[row][col] == exp_part:
-                nums = [row, col]
-                active_wells.append(nums)
-    return active_wells
+def _collect_exp_settings(abs_path, mp_file, delimiter, mp_dict):
```

Comment:
@stork119 This function extract information about optional experiment's parameters (i.e. stimulation.1.1), it parse both name and value of a given parameter for each map_plate well. The name is the same as the name of a similar function from old map_plate module.

abs_path = absolute_path
We're using this shortcut also in other modules (i.e. file_managment).